src/category/category.controller.ts

```diff
@@ -0,0 +1,39 @@
+import { Controller, Get, Post, Body, Delete, Param } from '@nestjs/common';
+import { CategoryService } from './category.service';
+import { CreateCategoryDto } from './dtos/create-category.dto';
+import { ApiTags, ApiResponse, ApiBody } from '@nestjs/swagger';
+
+@ApiTags('Categories')
+@Controller('categories')
+export class CategoryController {
+  constructor(private readonly categoryService: CategoryService) {}
+
```

Comment:
I hope that all endpoints can have decorators such 'ApiResponse or ApiBody' because I believe it might make your Swagger documentation look good.